### PR TITLE
Convert views/layouts Slim templates to ERB

### PR DESCRIPTION
**Pull request by spavlishak**

### CODE REVIEW

File: app/views/layouts/landing.html.erb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link href="/favicon.ico" rel="shortcut icon" />
+
```
Comment (yivo) [inline app/views/layouts/landing.html.erb]: @spavlishak where is viewport definition?

File: app/views/layouts/landing.html.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <%= stylesheet_link_tag 'application', media: 'all' %>
+
+    <%= include_gon %>
+    <%= javascript_include_tag "locales/#{I18n.locale}" %>
+    <%= javascript_include_tag 'application' %>
+
+    <%= yield(:scripts) if content_for?(:scripts) %>
+
+    <%= render 'shared/meta' %>
```
Comment (yivo) [inline app/views/layouts/landing.html.erb]: @spavlishak where is render shared/html5?

File: app/views/layouts/landing.html.erb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    <%= csrf_meta_tags %>
+  </head>
+
+  <body data-lang="<%= locale_name %>">
```
Comment (yivo) [inline app/views/layouts/landing.html.erb]: @spavlishak body id is missing

File: app/views/layouts/landing.html.erb [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    <%= render('shared/header') %>
```
Comment (yivo) [inline app/views/layouts/landing.html.erb]: @spavlishak don't write brackets when calling "render"

File: app/views/layouts/landing.html.erb [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  <head>
+    <link href="/favicon.ico" rel="shortcut icon" />
+
+    <%= stylesheet_link_tag 'application', media: 'all' %>
+
+    <%= include_gon %>
+    <%= javascript_include_tag "locales/#{I18n.locale}" %>
+    <%= javascript_include_tag 'application' %>
+
+    <%= yield(:scripts) if content_for?(:scripts) %>
```
Comment (yivo) [inline app/views/layouts/landing.html.erb]: @spavlishak where did you get it from? there is not such code in slim template

File: app/views/layouts/landing.html.erb [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    <%= csrf_meta_tags %>
+  </head>
+
+  <body data-lang="<%= locale_name %>">
+    <%= render('shared/header') %>
+
+    <div id="particles"></div>
+    <%= render 'shared/alert' %>
+
+    <div class="main container">
```
Comment (yivo) [inline app/views/layouts/landing.html.erb]: @spavlishak body's internas are totally invalid

File: app/views/layouts/admin.html.erb [hunk 7]
```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <%= stylesheet_link_tag :admin, media: 'all' %>
+    <%= javascript_include_tag :admin %>
+    <%= csrf_meta_tags %>
+  </head>
+
+  <body id="<% body_id %>">
```
Comment (yivo) [inline app/views/layouts/admin.html.erb]: @spavlishak missing `=` so this will not output anything

File: app/views/layouts/admin.html.erb [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+    <nav id="top-menu" class="navbar navbar-inverse" role="navigation">
+      <div class="container">
+        <ul class="nav navbar-nav navbar-left">
+          <li class="<%= check_active('dashboard') %>">
+            <%= link_to t('header.dashboard'), main_app.admin_dashboard_path %>
+          </li>
+
+          <% if can? :manage, Document %>
+            <li class="<%= check_active(Document) %>">
+              <%= link_to t(".menus.items.operating.documents"), admin_documents_path %>
```
Comment (yivo) [inline app/views/layouts/admin.html.erb]: @spavlishak use single quotes

File: app/views/layouts/admin.html.erb [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+
+          <% if can? :read, Proof %>
+            <li class="<%= check_active(Proof) %>">
+              <%= link_to t(".menus.items.operating.proofs"), admin_proofs_path %>
+            </li>
+          <% end %>
+
+          <% if can? :menu, Deposit %>
+            <li class="dropdown <%= check_active('deposits') %>">
+              <%= link_to 'javascript:;', class: 'dropdown-toggle', 'data-toggle' => 'dropdown' do %>
```
Comment (yivo) [inline app/views/layouts/admin.html.erb]: @spavlishak don't combine strings and symbols

File: app/views/layouts/admin.html.erb [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+                    <li><%= link_to c.title_text, [:admin, c.kls] %></li>
+                  <% end %>
+                <% end %>
+              </ul>
+            </li>
+          <% end %>
+
+          <% if can? :menu, Withdraw %>
+            <li class="dropdown <%= check_active('withdraws') %>">
+              <%= link_to 'javascript:;', class: 'dropdown-toggle', 'data-toggle' => 'dropdown' do %>
```
Comment (yivo) [inline app/views/layouts/admin.html.erb]: @spavlishak don't combine symbols & strings

File: app/views/layouts/admin.html.erb [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+                    <li><%= link_to c.title_text, [:admin, c.kls] %></li>
+                  <% end %>
+                <% end %>
+              </ul>
+            </li>
+          <% end %>
+
+          <% if can? :manage, Member %>
+            <li class="<% check_active('members') %>">
+              <%= link_to t(".menus.sections.members"), admin_members_path %>
```
Comment (yivo) [inline app/views/layouts/admin.html.erb]: @spavlishak always use single quotes in rb when you don't need interpolation

File: app/views/layouts/admin.html.erb [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+            </li>
+          <% end %>
+
+          <% if can? :manage, Ticket %>
+            <li class="<% check_active(Proof) %>">
+              <%= link_to t(".menus.items.operating.tickets"), admin_tickets_path %>
```
Comment (yivo) [inline app/views/layouts/admin.html.erb]: @spavlishak always use single quotes in rb when you don't need interpolation

File: app/views/layouts/admin.html.erb [hunk 13]
```diff
@@ -0,0 +1,106 @@
...
+            </li>
+          <% end %>
+
+          <li class="dropdown <%= check_active('statistic') %>">
+            <%= link_to 'javascript:;', class: 'dropdown-toggle', 'data-toggle' => 'dropdown' do %>
```
Comment (yivo) [inline app/views/layouts/admin.html.erb]: @spavlishak don't combine strings & symbols

File: app/views/layouts/api_v2.html.erb [hunk 14]
```diff
@@ -0,0 +1,59 @@
...
+  <head>
+    <%= include_gon %>
+    <%= stylesheet_link_tag 'api_v2', media: 'all' %>
+    <%= javascript_include_tag 'api_v2' %>
+    <%= render 'shared/html5' %>
+    <%= render 'shared/meta' %>
+  </head>
+
+  <body id="<%= body_id %>" data-lang="<%= locale_name %>">
+    <div class="navbar navbar-light navbar-static-top" role="navigation">
```
Comment (yivo) [inline app/views/layouts/api_v2.html.erb]: @spavlishak you have invalid class names, recheck please

Comment (yivo) [reply app/views/layouts/api_v2.html.erb]: .navbar.navbar-default.navbar-static-top role='navigation'

File: app/views/layouts/application.html.erb [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+
+    <%= csrf_meta_tags %>
+  </head>
+
+  <body data-lang="<%= locale_name %>" id="<%= body_id %>">
+    <%= content_for?(:header) ? yield(:header) : render('shared/header') %>
+
+    <div id="wrap">
+      <div class="container">
+        <%= render 'shared/alert' %>
```
Comment (yivo) [inline app/views/layouts/application.html.erb]: @spavlishak `= render 'shared/alert'` should be direct child of body

File: app/views/layouts/application.html.erb [hunk 16]
```diff
@@ -0,0 +1,35 @@
...
+    <%= yield(:scripts) if content_for?(:scripts) %>
+
+    <%= render 'shared/html5' %>
+    <%= render 'shared/meta' %>
+
+    <%= csrf_meta_tags %>
+  </head>
+
+  <body data-lang="<%= locale_name %>" id="<%= body_id %>">
+    <%= content_for?(:header) ? yield(:header) : render('shared/header') %>
```
Comment (yivo) [inline app/views/layouts/application.html.erb]: @spavlishak it should be in `#wrap`

File: app/views/layouts/application.html.erb [hunk 17]
```diff
@@ -0,0 +1,35 @@
...
+    <div id="wrap">
+      <div class="container">
+        <%= render 'shared/alert' %>
+        <%= content_for?(:frame) ? yield(:frame) : render('shared/frame') %>
+
+        <div id="push"></div>
+      </div>
+    </div>
+
+    <%= content_for?(:footer) ? yield(:footer) : render('shared/footer') %>
```
Comment (yivo) [inline app/views/layouts/application.html.erb]: @spavlishak it should be in `#wrap`. recheck whole template

**Review by yivo — CHANGES_REQUESTED** [review]

File: app/views/layouts/api_v2.html.erb (hunk 14 above)
Comment (spavlishak) [reply app/views/layouts/api_v2.html.erb]: that's because in bs4 navbar-default is navbar-light, and i was moving it to bs4

Comment (yivo) [reply app/views/layouts/api_v2.html.erb]: @spavlishak in this patch we don't add BS4. We convert slim to erb only.

**Review by yivo — APPROVED** [review]

**Pull request merged**